Source/CarthageKit/Xcode.swift

```diff
@@ -913,6 +917,47 @@ public func buildDependencyProject(dependency: ProjectIdentifier, rootDirectoryU
 		}
 }
 
+public func signingIdentities() -> SignalProducer<String, CarthageError> {
+	let securityTask = TaskDescription(launchPath: "/usr/bin/security", arguments: ["find-identity", "-v", "-p", "codesigning"])
+	
+	return launchTask(securityTask)
+		|> ignoreTaskData
+		|> mapError { CarthageError.TaskError($0) }
+		|> map { (data: NSData) -> String in
+			return NSString(data: data, encoding: NSStringEncoding(NSUTF8StringEncoding))! as String
+		}
+		|> flatMap(.Merge) { (string: String) -> SignalProducer<String, CarthageError> in
+			return string.linesProducer |> promoteErrors(CarthageError.self)
+		}
+		|> map { (identityLine: String) -> String? in
+			var error: NSError? = nil
+			let regex = NSRegularExpression(pattern: "\".+\"", options: nil, error: &error)
```

Comment:
I'd really like to see this split out into a private static variable, like `targetSettingsRegex` above.

It'd also be nice for it to be a bit more specific. I think something like this would work:

```
^\s*\d\)\s+\w+\s+"(.+)"$
```

Note the use of a capturing group for the actual name of the identity. Bonus points if you use `AllowCommentsAndWhitespace` to separate the comments to make it a little clearer. :smile: